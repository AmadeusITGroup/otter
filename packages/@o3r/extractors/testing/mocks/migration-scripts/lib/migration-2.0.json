{
  "version": "2.0.0",
  "changes": [
    {
      "contentType": "STYLE",
      "before": {
        "name": "old-lib-var"
      },
      "after": {
        "name": "new-lib-var2.0"
      }
    }
  ]
}
