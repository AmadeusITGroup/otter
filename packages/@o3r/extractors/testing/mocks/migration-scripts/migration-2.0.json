{
  "version": "2.0.0",
  "libraries": {
    "@o3r/my-lib": "3.0"
  },
  "changes": [
    {
      "contentType": "STYLE",
      "before": {
        "name": "old-var2"
      },
      "after": {
        "name": "new-var2"
      }
    }
  ]
}
