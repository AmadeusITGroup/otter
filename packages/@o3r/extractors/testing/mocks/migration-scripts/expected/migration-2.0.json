{
  "version": "2.0.0",
  "libraries": {
    "@o3r/my-lib": "3.0"
  },
  "changes": [
    {
      "contentType": "STYLE",
      "before": {
        "name": "old-var2"
      },
      "after": {
        "name": "new-var2"
      }
    },
    {
      "contentType": "STYLE",
      "before": {
        "name": "new-lib-var2.0"
      },
      "after": {
        "name": "new-lib-var2.5"
      }
    }
  ]
}
